docs/__common.soy

```diff
@@ -886,10 +886,10 @@ docsearch({
   {param desc}
   List of <a href="{ROOT}concept/build_target_pattern.html">build
   target patterns</a> that identify the build rules that can include
-  this rule as a dependency, for example, by listing it in 
+  this rule as a dependency, for example, by listing it in
   their <code>deps</code> or <code>exported_deps</code> attributes.
-  For more information, 
-  see {call buck.concept_link}{param page: 'Visibility' /}{param name: 'Visibility' /}{/call}.
+  For more information,
+  see {call buck.concept_link}{param page: 'visibility' /}{param name: 'Visibility' /}{/call}.
```

Comment:
Could you change "Visiblity" to lower case too?

Reply:
Sure.